500_Cluster_Admin/15_marvel.asciidoc

```diff
@@ -1,29 +1,27 @@
-
+[[marvel]]
 === Marvel for Monitoring
 
-At the very beginning of the book (<<marvel>>), we encouraged you to install
-Marvel,((("Marvel", "monitoring with")))((("clusters", "administration", "Marvel for monitoring"))) a management monitoring tool for Elasticsearch, because it would enable
-interactive code samples throughout the book.
-
-If you didn't install Marvel then, we encourage you to install it now.  This
-chapter introduces a large number of APIs that emit an even larger number
-of statistics.  These stats track everything from heap memory usage and garbage
-collection counts to open file descriptors.  These statistics are invaluable
-for debugging a misbehaving cluster.
+https://www.elastic.co/guide/en/marvel/current/index.html[Marvel] enables 
+you to easily monitor Elasticsearch through Kibana. You can view your 
+cluster’s health and performance in real time as well as analyze 
+past cluster, index, and node metrics. 
 
-The problem is that these APIs provide a single data point: the statistic
-_right now_.  Often you'll want to see historical data too, so you can 
-plot a trend.  Knowing memory usage at this instant is helpful, but knowing
-memory usage _over time_ is much more useful.
+While you can access a large number of statistics through the APIs described 
+in this chapter, they only show you what's going on at a single point in time.
+Knowing memory usage at this instant is helpful, but knowing
+memory usage _over time_ is much more useful. With Marvel, you can
+query and aggregate the metrics and visualize your cluster's 
```

Comment:
Should we rephrase this a bit?  Marvel queries/aggregates queries to visualize data, but doesn't really let *you* query the data (at least, not like Kibana).  Just don't want to accidentally setup false expectations. :)